crate/server/src/routes/google_cse/mod.rs

```diff
@@ -50,14 +50,21 @@ impl From<CseErrorReply> for HttpResponse {
     }
 }
 
-/// Get the status for Google CSE
+/// Get the status for Google CSE and the URL of the deployed KACLS (Key Access Control List Service)
 #[get("/status")]
 pub(crate) async fn get_status(
     req: HttpRequest,
     kms: Data<Arc<KMSServer>>,
+    cse_config: Data<Option<GoogleCseConfig>>,
 ) -> KResult<Json<operations::StatusResponse>> {
     info!("GET /google_cse/status {}", kms.get_user(&req)?);
-    Ok(Json(operations::get_status()))
+    let kacls_url =
+        <std::option::Option<crate::routes::google_cse::jwt::GoogleCseConfig> as Clone>::clone(
+            &cse_config,
+        )
+        .unwrap()
+        .kacls_url;
+    Ok(Json(operations::get_status(kacls_url)))
```

Comment:
IMHO, 2 things are wrong here: the hardcore `clone()` and the `unwrap()`.

Could we try something like this:

```suggestion
    let cse_config: &Option<GoogleCseConfig> = &cse_config.into_inner();
    let cse_config = cse_config.as_ref().ok_or_else(|| {
        KmsError::ServerError(
            "Unable to get a reference from as_ref of the Google CSE configuration"
                .to_string(),
        )
    })?;
    Ok(Json(operations::get_status(&cse_config.kacls_url)))
```

and change `get_status` to:

```
pub fn get_status(kacls_url: &str) -> StatusResponse {
```

Reply:
I can change to :
```
    let cse_config = cse_config.as_ref().clone().ok_or_else(|| {
        KmsError::ServerError(
            "Unable to get a reference from as_ref of the Google CSE configuration".to_string(),
        )
    })?;
    Ok(Json(operations::get_status(cse_config.kacls_url)))
```

But I think I still need the clone - and cse_config in get_status parameter function